class.plugin-settings.php

```diff
@@ -236,14 +236,24 @@ private function __construct()
             ),
             'map_tile_url_subdomains' => array(
                 'display_name'=>__('Map Tile URL Subdomains', 'leaflet-map'),
-                'default'=>'abc',
+                'default'=>'',
                 'type' => 'text',
                 'helptext' => sprintf(
                     '%1$s %2$s <br/> <code>[leaflet-map subdomains="1234"]</code>',
                     __('Some maps get tiles from multiple servers with subdomains such as a,b,c or 1,2,3 or cache1,cache2; pass a string where each char is its own subdomain, or pass comma-separated string', 'leaflet-map'),
                     $foreachmap
                 )
             ),
+            'map_tile_maxzoom' => array(
+                'display_name'=>__('Map Tile Max Zoom', 'leaflet-map'),
+                'default'=>'18',
+                'type' => 'number',
+                'helptext' => sprintf(
+                    '%1$s %2$s <br/> <code>[leaflet-map map_tile_maxzoom="1234"]</code>',
```

Comment:
probably should use a better default maxzoom than "1234"; sorry I missed this on my first pass